src/likelihoods/gaussian.jl

```diff
@@ -47,5 +47,5 @@ function (l::HeteroscedasticGaussianLikelihood)(f::AbstractVector{<:Real})
 end
 
 function (l::HeteroscedasticGaussianLikelihood)(fs::AbstractVector)
-    return MvNormal(first.(fs), l.invlink.(last.(fs)))
+    return MvNormal(first.(fs), l.invlink.(last.(fs)) * I)
```

Comment:
This should be:
```suggestion
    return MvNormal(first.(fs), Diagonal(l.invlink.(last.(fs)).^2))
```

Reply:
Actually it should be the opposite. We generally want to model the variance in heteroscedastic models, typically with `exp(f)`, `f^2`, or `var_max * logistic(f)`. One should probably change the one for the one-dimensional case

Reply:
Oh but you're right about the Diagonal